rconweb/api/audit_log.py

```diff
@@ -65,6 +61,7 @@ def wrapper(func):
 
 @csrf_exempt
 @login_required()
+@permission_required("api.can_view_audit_logs_autocomplete", raise_exception=True)
```

Comment:
oh wow you bothered typing it all? I'd have just done that automatically by inspecting the function name that is being decorated

Reply:
I thought about that, and it was a lot of copy/pasting around, but I wanted it to be searchable for people who might be not very familiar with the code base but want to mess with their permissions groups.